### PR TITLE
ftMasterHand

### PR DESCRIPTION
**Pull request by BR-**

### CODE REVIEW

**Comment by PsiLupan** [comment]
Go ahead and put these in a ftMasterHand folder when you get the chance, since I'll ask for that later.

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ ifneq ($(findstring MSYS,$(shell uname)),)
   WINDOWS := 1
 endif
 
-GENERATE_MAP ?= 0
+GENERATE_MAP ?= 1
```
Comment (r-burns) [inline Makefile:8]: don't forget to disable this before merging ;)

File: src/melee/ft/chara/ftMasterHand/ftMasterHand_22.c [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+
+
+// 801548D8 1514B8
+// https://decomp.me/scratch/rGeOV
+void lbl_801548D8(HSD_GObj* gobj) {
+    Fighter* r31_ft_userdata = gobj->user_data;
+    // f32 temp_f1;
+
+    // r31_ft_userdata->x2364 -= 1.0f;
+    if (
```
Comment (PsiLupan) [inline src/melee/ft/chara/ftMasterHand/ftMasterHand_22.c:307]: Please fix this spacing and just do:
```
if (--r31_ft_userdata->x2364 <= 0.0f && r31_ft_userdata->x2200_ftcmd_var0)
```

File: src/melee/ft/chara/ftMasterHand/ftMasterHand_25.c [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    func_80085134(gobj);
+    func_8015BE40(gobj, &ft->x234C_pos, &ft->x2358_stateVar7, attr->x2C, attr->x28);
+}
+
+
+
+
+// 80154E74 151A54
+// https://decomp.me/scratch/0wwnm
+void lbl_80154E74(HSD_GObj* gobj) {
```
Comment (PsiLupan) [inline src/melee/ft/chara/ftMasterHand/ftMasterHand_25.c:48]: Just put a ```return;```

File: src/melee/ft/chara/ftMasterHand/ftMasterHand_27.c [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+// 80155170 151D50
+void lbl_80155170(HSD_GObj* gobj) {
+    func_80085134(gobj);
+}
+
+
+
+// 80155190 151D70
+void lbl_80155190(HSD_GObj* gobj) {
```
Comment (PsiLupan) [inline src/melee/ft/chara/ftMasterHand/ftMasterHand_27.c:48]: Another ```return;```

File: src/melee/ft/chara/ftMasterHand/ftMasterHand_28.c [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+// 801552D4 151EB4
+void lbl_801552D4(HSD_GObj* gobj) {
+    func_80085134(gobj);
+}
+
+
+
+// 801552F4 151ED4
+void lbl_801552F4(HSD_GObj* gobj) {
```
Comment (PsiLupan) [inline src/melee/ft/chara/ftMasterHand/ftMasterHand_28.c:37]: Another ```return;```

**Review by PsiLupan — CHANGES_REQUESTED** [review]
Mostly just want you to get ```return;``` in the stub functions so they aren't just single line to be explicit about what they're doing.

**Comment by BR-** [comment]
`gawk -i inplace '/{$/ { empty = 1; print; next; } /^}$/ { if (empty == 1) print("    return;"); } 1 { empty = 0; print }' src/melee/ft/chara/ftMasterHand/ftMasterHand_*.c`

**Pull request merged**